src/scripts/accesskit/visible_alt_text.js

```diff
@@ -0,0 +1,46 @@
+import { descendantSelector, keyToCss } from '../../util/css_map.js';
+import { addStyle, removeStyle } from '../../util/interface.js';
+import { onPostsMutated } from '../../util/mutations.js';
+
+let imageBlockSelector;
+let css;
+
+const className = 'accesskit-visible-alt-text';
+
+const processImages = function () {
+  [...document.querySelectorAll(`${imageBlockSelector}:not(.${className})`)]
```

Comment:
`keyToCss` can't be relied on to produce just a single class selector. If Tumblr adds a new element with a different `imageBlock` class, this will end up as `.nonsenseOne, .nonsenseTwo:not(.accesskit-visible-alt-text)`, which won't work as intended.

Since there's no way to group selectors in CSS to achieve the desired effect with `:not()`, we can either use `keyToClasses` to only grab one classname to use, or we can do the more robust thing: filter the array instead of trying to construct a perfect selector.

```suggestion
  [...document.querySelectorAll(imageBlockSelector)]
    .filter(imageBlock => imageBlock.classList.contains(className) === false)
```

I believe it is technically more performant to use a perfect selector (browsers' CSS engines are *very* fast), but robustness has to come first.

Reply:
Ooh makes sense! I guess if we really wanted to stick with CSS selectors, maybe generating a string with keyToClasses and 
```imageBlockSelector = imageBlockClasses.foreach(imageBlockClass => `${imageBlockClass}:not(.${className})`).join(', ')``` would work? Though idk if the tradeoff in readability/simplicity is worth it.

Reply:
Readability comes first, especially when the loss in performance is negligible in real-life measures. `onPostsMutated` listener triggering is debounced, after all.

I admit I do want to write some kind of function that makes creating selectors like this more feasible, but my ideas for that are still cooking.